Performance.md

```diff
@@ -481,6 +481,31 @@ Sometimes TypeScript's type-checking can be impacted by computationally intensiv
 This is rare, but can happen.
 Upgrading to a newer version of TypeScript (which can be more efficient) or to a newer version of an `@types` package (which may have reverted a regression) can often solve the issue.
 
+## Performance Tracing
+
+You might find that these diagnostic methods are too opaque to get a sense of why TypeScript feels slow.
```

Comment:
I don't know what "opaque" means in this context, but it feels like "vague" or "imprecise" might be more appropriate?

Reply:
```suggestion
In some cases, the approaches above might not give you enough insight to understand why TypeScript feels slow.
```